polkadot/node/network/statement-distribution/src/v2/mod.rs

```diff
@@ -219,10 +229,17 @@ struct PerSessionState {
 	// getting the topology from the gossip-support subsystem
 	grid_view: Option<grid::SessionTopologyView>,
 	local_validator: Option<LocalValidatorIndex>,
+	// `true` if v2 candidate receipts are allowed by the runtime
+	v2_receipts: bool,
```

Comment:
can't we use `v2_receipts_allowed` here?

Reply:
Good point to make it consistent, I will go for `allow_v2_descriptors` used in other parts of the code